manifests/d/DuckStudio/FufuDevTools/2024.09.08.2000/DuckStudio.FufuDevTools.installer.yaml

```
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.5-1-22621-4111.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DuckStudio.FufuDevTools
+PackageVersion: 2024.09.08.2000
+InstallerLocale: zh-CN
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /verysilent /suppressmsgboxes
+  SilentWithProgress: /verysilent
+UpgradeBehavior: install
+Commands:
+- ffdev
+Protocols:
+- https
+AppsAndFeaturesEntries:
+- DisplayVersion: 2024.08.01.0000
```

```suggestion
- DisplayVersion: 2024.09.08.2000
```

I didn't notice this just now.